cocos/3d/lights/deprecated.ts

```diff
@@ -57,6 +58,13 @@ export { SpotLight as SpotLightComponent };
 cclegacy.SpotLightComponent = SpotLight;
 js.setClassAlias(SpotLight, 'cc.SpotLightComponent');
 
+/**
+ * Alias of [[PointLight]]
+ * @deprecated Since v1.2
```

Comment:
Deprecated since v1.2? It is a new deprecated function.

Reply:
sure